### ISSUE TITLE
Attendee - Event Page - Event Page Additions

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#111]
## Background
There are some additions to the Event Show page that we need to include. The scrolling and rounded edge of content will be taken care of with #99. This ticket is for the additional sections. 

## Scenario: A user navigates to an events page
- [ ] When I land on an Event page
- [x] Then I see the following above the Event Description
	- [x] The Event Time and Location (Link) (with icon)
	- [x] Other Performing Artists (with icon)
	- [x] Any Age Restrictions (with icon)
- [x] And the Event Description now has a Calendar icon to the left with the correct date
- [x] And directly underneath the Event Description there are two buttons:
	- [x] I'm Interested
	- [x] Share Event
- [x] And the "Tickets From" section is removed, making the People section full-width

Mock:
![event details](https://user-images.githubusercontent.com/6877299/45775274-e3ae0300-bc14-11e8-9acf-0834b97b5e9b.png)

**Comment by Mcscott142** [comment]
Moving into Week 38 commitment to replace #102

**Comment by brittanygay** [comment]
Starting this next

**Comment by brittanygay** [comment]
**Update**:
I've gotten through the majority of the edits needed. Going to take tomorrow to finish everything up.

**Comment by brittanygay** [comment]
My branch is in review right now. Once my PR is merged this will be ready for QA.

**IMPORTANT NOTE**
1. The scrolling and rounded edge of content will be taken care of with #99. We need to update that AC to include additional changes to the header that are shown in the new mock.
2. We also need to make a separate ticket to update the styling of the "Get Tickets" button section at the bottom

CC @Mcscott142

**Comment by brittanygay** [comment]
My PR has been approved and merged. More info and screenshots here: https://github.com/revelrylabs/bn-mobile-react/pull/113

Once updates are deployed this will be ready for QA @Mcscott142. Please note my comment above to keep in mind while viewing :)
